### PR TITLE
0.1.0

### PR DESCRIPTION
**Pull request by redcatbear**
Features:
* #1: Support asserting System.exit(int)

### CODE REVIEW

**Comment by redcatbear** [comment]
@kaklakariada, please review.

File: src/main/java/org/itsallcode/junit/sysextensions/ExitGuard.java [hunk 1]
```diff
@@ -1,36 +1,62 @@
...
-        } catch (final ExitSecurityException e)
-        {
-            assertEquals("Expected exit status code", expectedExitCode, e.getExitStatus());
-            return;
-        }
-        fail("Lambda did not cause a system exit as expected.");
-    }
-
     @Override
     public void postProcessTestInstance(final Object arg0, final ExtensionContext context) throws Exception
```
Comment (kaklakariada) [inline src/main/java/org/itsallcode/junit/sysextensions/ExitGuard.java]:
```suggestion
    public void postProcessTestInstance(final Object testInstance, final ExtensionContext context)
```

File: src/test/java/TestSystemExit.java [hunk 2]
```diff
@@ -1,40 +1,95 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+import static org.itsallcode.junit.AssertExit.assertExit;
```
Comment (kaklakariada) [inline src/test/java/TestSystemExit.java]: Don't leave this in the default package, move to org.itsallcode.junit.sysextensions

File: LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (kaklakariada) [inline LICENSE.txt:1]: There are two LICENSE files

File: pom.xml [hunk 4]
```diff
@@ -1,29 +1,75 @@
...
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.itsallcode</groupId>
+    <artifactId>junit5-system-extensions</artifactId>
+    <version>0.1.0</version>
+    <name>JUnit5 System Extensions</name>
+    <description>These extensions help testing behavior that is related to `System.x` calls like `exit(int).</description>
+    <properties>
+        <java.version>1.8</java.version>
+        <junit.version>5.3.1</junit.version>
+        <junit.platform.version>1.3.1</junit.platform.version>
```
Comment (kaklakariada) [inline pom.xml:12]: Properties are not used?
Dependencies seem to be managed via Eclipse Classpath, not via maven pom.xml

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: LICENSE.txt (hunk 3 above)
Comment (redcatbear) [reply LICENSE.txt:1]: Removed the duplicate. I am thinking: should this module be EPL2 like JUnit instead?

File: src/test/java/TestSystemExit.java (hunk 2 above)
Comment (redcatbear) [reply src/test/java/TestSystemExit.java]: Done.

File: pom.xml (hunk 4 above)
Comment (redcatbear) [reply pom.xml:12]: This is really strange. Investigating...
Okay, you were right. I fixed that. Strangely enough I could build and test outside of Eclipse too.

**Comment by redcatbear** [comment]
@kaklakariada, please check again.

**Review by kaklakariada — APPROVED** [review]

File: LICENSE.txt (hunk 3 above)
Comment (kaklakariada) [reply LICENSE.txt:1]: No idea. Whatever you prefer.

**Pull request merged**